### PR TITLE
Adding Test cases for JIRA-DS1103

### PR DESCRIPTION
**Pull request by madhawa-gunasekara**

### CODE REVIEW

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import org.wso2.dss.integration.common.utils.DSSTestCaseUtils;
+import org.wso2.dss.integration.test.DSSIntegrationTest;
+
+import javax.activation.DataHandler;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This test was written to verify the fix for https://wso2.org/jira/browse/DS-1103
```
Comment (lafernando) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java]: End the Java doc comment sentence with a period.

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ */
+public class DS1103DuplicateDataServiceTestCase extends DSSIntegrationTest {
+
+	private final String serviceName = "H2SimpleJsonTest";
+	private List<File> sqlFileLis;
+
+	@BeforeClass(alwaysRun = true)
+	public void serviceDeployment() throws Exception {
+
```
Comment (lafernando) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java]: Remove unneeded whitespace.

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java (hunk 1 above)
Comment (madhawa-gunasekara) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java]: fixed with 12d4ea4

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java (hunk 2 above)
Comment (madhawa-gunasekara) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/dss/integration/test/jira/issues/DS1103DuplicateDataServiceTestCase.java]: fixed with 12d4ea4

**Comment by lankavitharana** [comment]
pull request merged

**Pull request merged**